src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -19,6 +19,12 @@
             [showResult]="showResults && currentResult !== null"
         >
         </jhi-exercise-details-student-actions>
+        <button class="dev-button" id="submission" *ngIf="!isInProduction() && hasNoLocalSetup()" (click)="simulateSubmission()">
```

Comment:
Since ids need to be globally unique in the HTML of a page, this name might be a bit risky. Same for `result` below.

Reply:
i removed the id, since i don't use it anymore